### PR TITLE
Bngp 1642 summary cya flow

### PR DESCRIPTION
**Pull request by shaundefra**

### CODE REVIEW

File: src/features/page_objects/check-and-submit.page.js [hunk 1]
```diff
@@ -5,60 +5,85 @@ class CheckAndSubmit extends Page {
...
-    return $("#add-habitat-information");
+  get emailAddressValue(){
+    return $("/html/body/div[1]/main/form/div/div/div/dl[1]/div[3]/dd[1]");
   }
 
-  get addHmmp(){
-    return $("#add-habitat-management");
+
+  get landBoundaryTypeValue(){
+    return $("/html/body/div[1]/main/form/div/div/div/dl[2]/div[1]/dd[1]/span");
```
Comment (timstone) [inline src/features/page_objects/check-and-submit.page.js:26]: .govuk-summary-list__row:nth-child(1) span

File: src/features/page_objects/check-and-submit.page.js [hunk 2]
```diff
@@ -5,60 +5,85 @@ class CheckAndSubmit extends Page {
...
+  }
+
+  get landBoundaryFilenameValue(){
+    return $("/html/body/div[1]/main/form/div/div/div/dl[2]/div[2]/dd[1]/span");
```
Comment (timstone) [inline src/features/page_objects/check-and-submit.page.js:30]: .govuk-summary-list__row+ .govuk-summary-list__row span

File: src/features/page_objects/check-and-submit.page.js [hunk 3]
```diff
@@ -5,60 +5,85 @@ class CheckAndSubmit extends Page {
...
-  get addMetric() {
-    return $("#add-habitat-information");
+  get emailAddressValue(){
+    return $("/html/body/div[1]/main/form/div/div/div/dl[1]/div[3]/dd[1]");
   }
 
-  get addHmmp(){
-    return $("#add-habitat-management");
+
+  get landBoundaryTypeValue(){
```
Comment (timstone) [inline src/features/page_objects/check-and-submit.page.js:25]: You have good selectors now [data-testid='land-boundary-file-type-value']

**Review by timstone — CHANGES_REQUESTED** [review]
Selecta!

File: src/features/page_objects/check-and-submit.page.js (hunk 3 above)
Comment (shaundefra) [reply src/features/page_objects/check-and-submit.page.js:24]: updated the datatestids that are present now.
new ones to be added in upcoming ticket https://eaflood.atlassian.net/browse/BNGP-1959

**Pull request merged**